.eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
         '@typescript-eslint/explicit-function-return-type': 'off',
         'react/prop-types': 'off',
         '@typescript-eslint/no-object-literal-type-assertion': 'warn',
+        '@typescript-eslint/camelcase': 'off',
```

Comment:
is this  for the `TODO_`?

Reply:
yes, the rule seems a bit overzealous?

Reply:
perhaps, but if it's for a TODO. I'd prefer we just add an inline exclusion.

Reply:
it'd be one inline comment per usage and i don't wanna discourage this pattern in general, it's a good way to denote things that are a bit out of the ordinary and it grabs attention

Reply:
maybe warn would be better suited in that case?